README.md

```diff
@@ -1 +1,165 @@
-# Uptasticsearch
\ No newline at end of file
+# uptasticsearch
+
+## Introduction
+
+This project tackles the issue of getting data out of Elasticsearch and into a tabular format in R.
+
+# Table of contents
```

Comment:
We can add the version stuff in a follow-up PR once we've tested on many versions. 

But yeah, agreed on adding a note about auth